ai/nl_agent.py

```
@@ -0,0 +1,284 @@
+            request_id=req_id,
+            intent=IntentType.LEARN_CONCEPT,
+            text="Ask me about specific tactics like forks, pins, or skewers for a detailed explanation.",
+            confidence=0.5,
+        )
+
+    @staticmethod
+    def _fmt_eval(evaluation: float | None, complexity: ComplexityLevel) -> str:
+        if evaluation is None:
+            return "unknown"
+        if complexity == ComplexityLevel.BEGINNER:
+            if evaluation > 1.0:   return "clearly better for white"
```

Ruff (0.15.11)

[error] Multiple statements on one line (colon)

(E701)